### DISCUSSION TITLE
Save Existing content without Overwrite

### DISCUSSION THREAD

**Discussion by IkelAtomig** [coddingtonbear/obsidian-local-rest-api#52] [Q&A]
For example, I selected a portion of a paragraph in a page and I save it. Now, I select another paragraph from the same page. Instead of over-writing the existing content. How do I append it as extra data ?

**Comment by coddingtonbear** [comment]
It really depends on what kind of change you want to make.  If you're just hoping to _append_ to that note, you probably want to use the `POST` method, but you should look around at the online docs to see what options exist and which most closely aligns with your goals.
